whelktool/scripts/cleanups/2019/11/lxl-2780-remove-properties-from-topics/script.groovy

```diff
@@ -0,0 +1,39 @@
+/*
+See LXL-2780 & LXL-2395 for more info.
+*/
+
+PrintWriter failedIDs = getReportWriter("failed-to-delete-bibIDs")
+scheduledForChange = getReportWriter("scheduledForChange")
+
+selectBySqlWhere("""collection = 'auth' AND
+ data#>>'{@graph,1,@type}' = 'Geographic' OR
+ data#>>'{@graph,1,@type}' = 'Topic' OR
+ data#>>'{@graph,1,@type}' = 'TopicSubdivision' OR
+ data#>>'{@graph,1,@type}' = 'Temporal' OR
+ data#>>'{@graph,1,@type}' = 'ComplexSubject'""", silent: false) { documentItem ->
```

Comment:
Doesn't AND bind stronger than OR? So the potential filter speedup is mitigated if the query here means (auth and Geographic) or ..."?